### ISSUE TITLE
Objective

### ISSUE THREAD

**Issue by RuanAragao** [RuanAragao/calculator#3]
Please, read the [README](https://github.com/RuanAragao/calculator/blob/main/README.md). Thanks!

**Comment by RuanAragao** [comment]
> Does it mean creating a new calculator for different calculations?

Yes, a new calculator.

Create a calculator however you see fit, it can be a standard one with basic operations, or with a specific application. The important thing is to function according to its purpose.

Example: a compound interest calculator.

**Comment by harmit17** [comment]
Understood. I can work on it. You do not need to assign this to me, right? I can just start working right away.

**Comment by RuanAragao** [comment]
> Understood. I can work on it. You do not need to assign this to me, right? I can just start working right away.

Right. You can open an issue for your work. Or if you prefer, you can just send the PR.

**Comment by RuanAragao** [comment]
This issue is just a warning to read the README

> Please, read the [README](https://github.com/RuanAragao/calculator/blob/main/README.md). Thanks!
